runscanner/src/main/java/uk/ac/bbsrc/tgac/miso/runscanner/processors/DefaultIllumina.java

```diff
@@ -219,18 +219,23 @@ public NotificationDto process(File runDirectory, TimeZone tz) throws IOExceptio
       }
 
       if (!updatedHealth.isPresent() && dto.getNumReads() > 0) {
-        // Well, that didn't work. Maybe there are netcopy files.
-        long netCopyFiles = IntStream.rangeClosed(1, dto.getNumReads())//
-            .mapToObj(read -> String.format("Basecalling_Netcopy_complete_Read%d.txt", read))//
-            .map(fileName -> new File(runDirectory, fileName))//
-            .filter(File::exists)//
-            .count();
-        if (netCopyFiles == 0 && !new File(runDirectory, "CopyComplete.txt").exists()) {
-          // This might mean incomplete or it might mean the sequencer never wrote the files
+        if (new File(runDirectory, "CopyComplete.txt").exists()) {
+          // It's allegedly done.
+          updatedHealth = Optional.of(HealthType.Completed);
```

Comment:
Should there be another `completeness_method_success` label for this?

Reply:
Yes. I'll do that.